xontrib/voxapi.py

```diff
@@ -32,16 +32,68 @@ def __init__(self):
         else:
             self.venvdir = builtins.__xonsh_env__['VIRTUALENV_HOME']
 
-    def create(self, name):
+    def create(self, name, *, system_site_packages=False, symlinks=False, 
+               with_pip=True):
         """Create a virtual environment in $VIRTUALENV_HOME with python3's ``venv``.
 
         Parameters
         ----------
         name : str
             Virtual environment name
+        system_site_packages: bool
```

Comment:
These should be in numpydoc format which requires a space on both sides of the colon